projects/spectator/test/ngonchanges-input/ngonchanges-input.component.ts

```diff
@@ -0,0 +1,22 @@
+import { ChangeDetectionStrategy, Component, Input } from '@angular/core';
+
+@Component({
+  selector: 'app-root',
+  standalone: true,
+  template: `
+    <button [disabled]="btnDisabled">
+      {{ btnText }}
+    </button>
+  `,
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class NgOnChangesInputComponent {
```

Comment:
```suggestion
export class OnPushComponent {
```
I think thats more to the point.

Reply:
Correct, but let's not wait for this fix